addons/sriov-device-plugin/enable

```diff
@@ -142,16 +142,15 @@ def _enable_sriovdp(resources: dict) -> None:
     while time.time() - now <= timeout:
         sriov_allocatable_resources = _get_sriov_allocatable_resources()
 
-        for resource, pci_addresses in resources.items():
-            if sriov_allocatable_resources[f"intel.com/{resource}"] != str(len(pci_addresses)):
-                print("SR-IOV Network Device Plugin not ready, " +
-                    f"intel.com/{resource} does not have enough allocatable resources.")
-                break
-        else:
+        if all(
+            sriov_allocatable_resources.get(f"intel.com/{resource}") == str(len(pci_addresses))
+            for resource, pci_addresses in resources.items()
+        ):
```

Comment:
nice, this looks clean.

Though, the previous implementation (with ``for``) had the benefit of logging what resources were missing, making it a bit more transparent of what it is waiting for and what exactly is the issue if it's not succeeding. If I am to use this, and if something goes wrong, this will eventually timeout (raising the `TimeoutError("Unable to start SR-IOV Network Device Plugin. Operation timed out!")`) but I couldn't guess why it timed out, or if it's because some resources were missing.

Reply:
I reverted to the original for loop to preserve info about missing resource. The only change now is using `get()`.

Reply:
I reverted to the original for loop to preserve info about missing resource. The only change now is using `get()`.